src/components/wizard-internet/wizard-internet.js

```diff
@@ -8,17 +8,128 @@ export default module('app.components.wizard-internet', [])
     },
     // TODO: handle vpn
     controller: class WizardContactCtrl {
+
       constructor($scope) {
         'ngInject';
 
         this.newInternet = {};
+        this.internetTunnelEnabled = false;
+        this.meshTunnelEnabled = false;
```

Comment:
it's not wrong to initialize these here but we don't need to. They're `undefined` by default and `updateFromInput()` will set them anyway.